.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
 
       - name: Clone CCI
         run: |
-          git clone https://qchateau:$GITHUB_TOKEN@github.com/conan-center-bot-testing/conan-center-index.git
+          git clone https://${{ github.repository_owner }}:$GITHUB_TOKEN@github.com/${{ github.repository_owner }}/conan-center-bot-dev.git conan-center-index
```

Comment:
You are probably not ok with this change (my change), because github does not seem to allow PRs between two repos if one is not a github fork of the other in the first place. The fact that their branches have a comon ancestor is not sufficient. The problem is that you can fork a given repository only once for a given github user.
I think the real solution to this would be to use `https://${{ github.repository_owner }}:$GITHUB_TOKEN@github.com/${{ github.repository_owner }}/conan-center-index.git` for both workflows, but differentiate the branches by using a prefix `ccb/prod/` or `ccb/dev/`.

Reply:
Using a prefix is a good idea.
Or changing the code to just not push the branches in dev. It's been a long time I've worked on this code but it should be possible

Reply:
Let's keep things separate: I'll make an other PR for changes allowing forks to work